app/models/problem.rb

```diff
@@ -85,25 +84,25 @@ def self.cache_notice(id, notice)
     user_agent_digest = Digest::MD5.hexdigest(notice.user_agent_string)
 
     Problem.where('_id' => id).find_one_and_update({
-      '$set' => {
-        'environment' => notice.environment_name,
-        'error_class' => notice.error_class,
-        'last_notice_at' => notice.created_at.utc,
-        'message' => notice.message,
-        'resolved' => false,
-        'resolved_at' => nil,
-        'where' => notice.where,
-        "messages.#{message_digest}.value" => notice.message,
-        "hosts.#{host_digest}.value" => notice.host,
-        "user_agents.#{user_agent_digest}.value" => notice.user_agent_string,
-      },
-      '$inc' => {
-        'notices_count' => 1,
-        "messages.#{message_digest}.count" => 1,
-        "hosts.#{host_digest}.count" => 1,
-        "user_agents.#{user_agent_digest}.count" => 1,
-      }
-    }, return_document: :after)
+                                                     '$set' => {
+                                                       'environment' => notice.environment_name,
+                                                       'error_class' => notice.error_class,
+                                                       'last_notice_at' => notice.created_at.utc,
+                                                       'message' => notice.message,
+                                                       'resolved' => false,
+                                                       'resolved_at' => nil,
+                                                       'where' => notice.where,
+                                                       "messages.#{message_digest}.value" => notice.message,
+                                                       "hosts.#{host_digest}.value" => notice.host,
+                                                       "user_agents.#{user_agent_digest}.value" => notice.user_agent_string,
+                                                     },
+                                                     '$inc' => {
+                                                       'notices_count' => 1,
+                                                       "messages.#{message_digest}.count" => 1,
+                                                       "hosts.#{host_digest}.count" => 1,
+                                                       "user_agents.#{user_agent_digest}.count" => 1,
+                                                     }
+                                                   }, return_document: :after)
```

Comment:
Are there any other options? This really ruins my 3-way diffs especially on small monitors and causes me to have to horizontally scroll to see the code.